README.md

```diff
@@ -1,5 +1,28 @@
-# loopback-connector-cos
+<h2>Instructions on how to use this loopback connector with COS + APIC </h2>
 
-IBM Cloud Object Storage connector for LoopBack
```

Comment:
Please preserve this description too, it matched `description` field in `package.json`.